literature-cleaned.bib

```diff
@@ -38,6 +38,15 @@ @inproceedings{KTSB:ICSE21
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2021 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+% seems to have no pages. Maybe just an online article?

```

Comment:
Sorry Michael, but I forgot to mention that literature-cleaned.bib is automatically generated from literature.bib.

Hence, you need to add all changes to literature.bib and also need to add addresses of publishers.

There is no need to edit literature-cleaned, as I/we can automatically generate the file afterwards.

Reply:
File still appears as changed here.

Reply:
@mnieke moved his changes to literature.bib. The cleaned file is still changed because it is re-generated.